src/NServiceBus.Core.Tests/AssemblyScanner/When_directory__with_non_dot_net_dll_is_scanned.cs

```diff
@@ -17,17 +14,10 @@ public class When_directory__with_non_dot_net_dll_is_scanned
         [SetUp]
         public void Context()
         {
-            var codeBase = Assembly.GetExecutingAssembly().CodeBase;
-            var uri = new UriBuilder(codeBase);
-            var path = Uri.UnescapeDataString(uri.Path);
-            var directoryName = Path.GetDirectoryName(path);
-
-            var testDllDirectory = Path.Combine(directoryName, "TestDlls");
-            var assemblyScanner = new AssemblyScanner(testDllDirectory)
+            var assemblyScanner = new AssemblyScanner(AssemblyScannerTests.GetTestAssemblyDirectory())
```

Comment:
I'm always very sceptic when I see a test which references another test. I know this is nitpicker corner but this usually fires an alarm in my reviewer brain ;) I know in that case this isn't really an issue but I want to point it out.